test/e2e/autotls/auto_tls_test.go

```diff
@@ -1,5 +1,3 @@
-// +build e2e
```

Comment:
We need this to guard TestTls from running when you say `go test ./...` right?

Reply:
This is an unexpected change.